contracts/strategy/BaseStrategy.sol

```diff
@@ -90,21 +93,17 @@ abstract contract BaseStrategy is IStrategy, AccessControl {
         address _vault,
         address _treasury,
         address _ethAnchorRouter,
-        address _exchangeRateFeeder,
+        AggregatorV3Interface _aUstToUstFeed,
         IERC20 _ustToken,
         IERC20 _aUstToken,
         uint16 _perfFeePct,
         address _owner
     ) {
         require(_ethAnchorRouter != address(0), "0x addr: _ethAnchorRouter");
-        require(
-            _exchangeRateFeeder != address(0),
-            "0 addr: _exchangeRateFeeder"
-        );
```

Comment:
why is the check not being done on the new `aUstToUstFeed`?

Reply:
if aUstToUstFeed is address(0). then aUstToUstFeed.decimals() cannot be called.

Reply:
I know, but that's fragile. if we later refactor and move that call to another function, this will no longer be validated

Reply:
sorry, I don't understand what you mean